core/src/com/gaalf/view/GameView.java

```diff
@@ -102,11 +103,13 @@ public void clearWindow(){
         nextLevelButton.setVisible(true);
         nextLevelButton.setDisabled(false);
         levelClearedWindow.remove();
-        getTable().add(pauseButton).right().padTop(20);
+        getTable().addActorAt(0, pauseButton);
+        getTable().getCell(pauseButton).padTop(20);
```

Comment:
Crashes here on NextLevel. Null pointer on the getCell(pauseButton)

Reply:
This will be fixed in #54 , where GameView and MPGameView methods are abstracted to the BaseGameView. If #54 is merged first that issue will resolve itself.